docs/reference/inference/service-azure-ai-studio.asciidoc

```diff
@@ -0,0 +1,172 @@
+[[infer-service-azure-ai-studio]]
+=== Azure AI studio {infer} `service`
```

Comment:
Just double checking that we do want these services pages to have `service` in a code block?

![image](https://github.com/elastic/elasticsearch/assets/56361221/335eabd1-1217-487a-b73c-17aa0bf4cc3c)

Reply:
Well, I think it's understandable without explicitly putting it into a code block, so I remove them.